adapter/src/main/java/com/twilio/kudu/sql/schema/BaseKuduSchemaFactory.java

```diff
@@ -39,8 +39,15 @@ public BaseKuduSchemaFactory(final Map<String, KuduTableMetadata> kuduTableConfi
   @Override
   public Schema create(SchemaPlus parentSchema, String name, Map<String, Object> operand) {
     final String connectString = (String) operand.get(KuduSchema.KUDU_CONNECTION_STRING);
-    return schemaCache.computeIfAbsent(connectString,
-        (masterAddresses) -> new KuduSchema(masterAddresses, kuduTableConfigMap, operand));
+    final Boolean disableSchemaCache = Boolean.valueOf((String) operand.get(KuduSchema.DISABLE_SCHEMA_CACHE));
```

Comment:
Since we add materialized views after the `KuduSchema` is first generated if it is cached the MVs won't be used, so add an option to disable caching.

Reply:
I am guessing this means we eventually need a new way to define KuduMVSchema so that those can be cached.

Reply:
This is primarily an issue with tests, where we are creating materialized views in the test. If views exist then the first time the KuduSchema is created it will create it. Need to think of a better way to handle this.